WISE-Community/WISE
# Restyled the homepage and teacher unit libraries

To test:
- Load homepage and make sure library units load correctly, search and filters work
- Log in as a teacher
- On teacher homepage, select the 'Browse WISE Units' tab
- Make sure all units load correctly for each of the three library sections ('WISE Tested', 'Community Built', 'My Units')
- Make sure search and filter works for all three library sections
- Make sure pagination works and updates/resets when search or filters change

Closes #1592.
